cinema-backend/src/test/java/pl/edu/pw/ee/cinemabackend/steps/UpdateMovieSteps.java

```diff
@@ -0,0 +1,132 @@
+import pl.edu.pw.ee.cinemabackend.entities.movie.Movie;
+import pl.edu.pw.ee.cinemabackend.entities.movie.interfaces.MovieRepository;
+import pl.edu.pw.ee.cinemabackend.entities.user.User;
+import pl.edu.pw.ee.cinemabackend.entities.user.UserRole;
+import pl.edu.pw.ee.cinemabackend.runners.SpringIntegrationTest;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+public class UpdateMovieSteps extends SpringIntegrationTest {
+
+    @Autowired
+    private MovieService movieService;
+    @Autowired
+    private MovieRepository movieRepository;
+    private MovieResponse movieResponse;
+    private MovieRequest movieRequest;
+    private User user;
+    private long Id;
+
+    @Given("^User is logged in with role (.+)$")
+    public final void userIsSignedIn(String role) {
+
+        UserRole userRole = UserRole.valueOf(role);
+        user = User.builder()
+                .username("kicia2@mail.com")
+                .name("John")
+                .surname("Doe")
+                .password("kicia.?312312312As")
+                .userRole(userRole)
+                .build();
+
+        Movie movie = Movie.builder()
+                .title("title")
+                .description("description")
+                .timeDuration("duration")
+                .minimalAge(12)
+                .build();
+        movie = movieRepository.saveAndFlush(movie);
+        Id = movie.getMovieId();
+
+        movieRequest = MovieRequest.builder()
+                .title("title")
+                .description("description")
+                .timeDuration("15")
+                .minimalAge(15)
+                .build();
+
+        Authentication authentication = new UsernamePasswordAuthenticationToken(user, null, user.getAuthorities());
+        SecurityContextHolder.getContext().setAuthentication(authentication);
+    }
+
+    @Given("^User is signed in with role ADMIN, submits (.+) Id title (.+), description (.+), duration (.+) and minimal age (.+)$")
+    public final void userIsSignedIn(String id, String title, String description, String duration, int age) {
+        title = checkIfStringIsNull(title);
+        description = checkIfStringIsNull(description);
+        duration = checkIfStringIsNull(duration);
+        user = User.builder()
+                .username("kicia2@mail.com")
+                .name("John")
+                .surname("Doe")
+                .password("kicia.?312312312As")
+                .userRole(UserRole.ADMIN)
+                .build();
+
+        if (id.equals("good")) {
+
+            Movie movie = Movie.builder()
+                    .title("title")
+                    .description("description")
+                    .timeDuration("duration")
+                    .minimalAge(12)
+                    .build();
+            movie = movieRepository.saveAndFlush(movie);
+            Id = movie.getMovieId();
+        } else {
+            Id = -2;
```

Comment:
Dodatkowo `Id` jest typu `long`, a tutaj przypisujesz `int` to robisz niejawną konwersję typów powinno być `-2L`